packages/synapse-react-client/src/components/widgets/facet-nav/PlotsContainer.tsx

```diff
@@ -257,7 +257,7 @@ export default function PlotsContainer(props: PlotsContainerProps) {
               showPlotVisualization ? '' : 'hidden'
             } ${showMoreButtonState === 'LESS' ? 'less' : ''}`}
           >
-            <div className="PlotNav__row" role="list">
+            <div className="PlotsContainer__row" role="list">
```

Comment:
This class was erroneously renamed, causing it to not get the appropriate styles from CSS